server/util/clickhouse/schema/schema.go

```diff
@@ -378,7 +378,7 @@ func (t *TestTargetStatus) TableName() string {
 }
 
 func (t *TestTargetStatus) TableOptions() string {
-	return fmt.Sprintf("ENGINE=%s ORDER BY (group_id, repo_url, commit_sha, label, invocation_uuid)", getEngine())
+	return fmt.Sprintf("ENGINE=%s ORDER BY (group_id, repo_url, commit_sha, label, invocation_uuid) SETTINGS deduplicate_merge_projection_mode = 'rebuild'", getEngine())
```

Comment:
Customers can use an earlier version of clickhouse. Can we check the version of the clickhouse? and only if it's greater than a certain version, we add this setting. 
For mysql, we used "select version()". [code](https://github.com/buildbuddy-io/buildbuddy/blob/224fabd3d645cdc0d7b5fe483ac927eb6b30d419/server/tables/tables.go#L998)